### DISCUSSION TITLE
Komunikacja pomiędzy subskrypcjami

### DISCUSSION THREAD

**Discussion by Michal-Witwicki** [AzureCommunityPL/group#206] [Pomoc] [answered]
Cześć, czy wie ktoś jak to jest z grupami zasobów pomiędzy różnymi subskrypcjami? Mam subskrypcje MSDN oraz MPN w każdej z nich utworzoną jedną maszynę w ramach indywidualnych grup zasobów. Zakładam, że tworząc kolejną maszynę powinienem móc niezależnie od tego której subskrypcji użyje, utworzyć VM np. w subskrypcji MPN w oparciu o grupę zasobów istniejącą w ramach MSDN. Niestety przy tworzeniu VM nie widzę zasobów z innej subskrypcji. Zastanawiam się czy robię coś źle czy tak to właśnie działa po prostu.

**Comment by miqm** [comment, marked as answer]
Nie da się, to jest struktura hierarchiczna. RG jest tworzone w konkretniej subskrypcji, zasoby są tworzone w RG.

Więcej info tutaj: https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-setup-guide/organize-resources?tabs=AzureManagementGroupsAndHierarchy

**Reply by Michal-Witwicki** [reply]
Rozumiem, dzięki za info. To jeszcze jedno szybkie pytanie, czy mając dwie różne RG w ranch dwóch różnych subskrypcji jestem w stanie sieciowo rozmawiać pomiędzy usługami w ramach tych dwóch RG/Subskrypcji?

**Reply by miqm** [reply]
Tak, ale musisz użyć mechanizmu vnet peering. Tutaj nawet Tenant Cię nie ogranicza - możesz zestawić połączenie sieciowe pomiędzy dowolnymi sieciami w Azure.

**Reply by kamil-mrzyglod** [reply]
Ja jeszcze tutaj może dodam - mając **resourceId** czyli cały identyfikator zasobu, na który składa się subskrypcja, RG, typ zasobu i jego nazwa z grubsza jesteś w stanie zidentyfikować dowolny zasób i połączyć się z nim tak długo, jak masz do niego dostęp. Czyli będąc tylko w jednej subskrypcji nie jesteś w stanie zarządzać zasobami w innej, w momencie jak dostaniesz dostęp do drugiej to automatycznie Twoje możliwości się poszerzają.

Połączenia sieciowe to inny temat, bo tak jak kolega @miqm napisał, tutaj z grubsza musisz wiedzieć do jakiej sieci się połączyć, nawet konkretnych dostępów nie musisz mieć. 

Mógłbyś widzieć swoje zasoby z innej subskrypcji zakładając, że są podpięte pod tego samego tenanta. Jeśli tak nie jest, każdorazowo musisz się przepinać się pomiędzy nimi bo w każdym tenancie(nawet jeśli logujesz się tym samym loginem) masz oddzielne konto.

**Reply by smajchrz** [reply]
> 
> 
> Tak, ale musisz użyć mechanizmu vnet peering. Tutaj nawet Tenant Cię nie ogranicza - możesz zestawić połączenie sieciowe pomiędzy dowolnymi sieciami w Azure.

Warto jeszcze dodać, że peering potrafi wygenerować spore koszty.
